.gitignore

```diff
@@ -8,3 +8,4 @@ _site
 # Generated by 'yarn dev'
 /_includes/2017/critical/*
 /assets/packed/*
+/.cache
```

Comment:
I don't know if adding the `.cache` directory to `.gitignore` was the right decision, because at the moment you're the only one who's having this problem. Maybe @rstacruz can decide whether to add it.

If end up adding it, it should be `.cache/` instead of `/.cache`, and it should be added below line 6.

Reply:
Yah right, updated according to your suggestion, Thanks  . but I'm facing this problem someone else might. and .gitignore should be universal.